data/tribes/initialization/empire/units.lua

```diff
@@ -1227,13 +1278,21 @@ wl.Descriptions():new_tribe {
       {
          name = "empire_stonemasons_house",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire production site: Stonemason's House
+            lore = pgettext("empire_building", "‘marmoris insania!’ (Marble Madness!)"),
+            -- TRANSLATORS: Lore author helptext for an empire production site: Stonemason's House
+            lore_author = pgettext("A Barabarian about them imperial marbel clumns"),
             -- TRANSLATORS: Purpose helptext for an empire production site: Stonemason's House
             purpose = pgettext("empire_building", "Carves marble columns out of marble.")
          }
       },
       {
          name = "empire_sawmill",
          helptexts = {
+            -- TRANSLATORS: Lore helptext for an empire production site: Sawmill
+            lore = pgettext("empire_building", "‘Veni vidi serram duceret’ (I came, I saw, and pulled the saw)"),
```

Comment:
@klaus-halfmann For the English pun? I don't think that's a good idea.

Reply:
Its actualy the german pun and I could not resist.
"Ich kam, sah und sägte" Oh and its originally greek: https://de.wikipedia.org/wiki/Veni,_vidi,_vici

Reply:
That quote is already used for the tower anyway.